config/settings.yml

```diff
@@ -1,7 +1,6 @@
 webdriver:
   browser: chrome
   headless: false
-
```

Comment:
@Seanhaoxiang - could we add this blank line back?  It helps to identify individual settings when different categories of settings are separated.